tests/integration/routes/test_view_submission.py

```diff
@@ -1,363 +0,0 @@
-import simplejson as json
```

Comment:
If we're removing the test for this, then we should probably remove the code that handles view submission as well.

Reply:
Just spoke to @ajmaddaford about this. Suggestion is to leave the view submission feature in runner. So in that case, we should leave the tests here as well.

Reply:
Have put this file back and removed references to the thank you page.